web/src/colors.ts

```diff
@@ -92,4 +93,25 @@ export const AMINOACID_COLORS: Record<string, string> = {
   'Z': '#AAAAAA',
   '*': '#AAAAAA',
   '-': '#AAAAAA',
+
+  'Ala': '#EAEABA',
+  'Val': '#EAEA9F',
+  'Leu': '#E1E177',
+  'Ile': '#C9C94D',
+  'Cys': '#E3F9B0',
+  'Asp': '#E98F6D',
+  'Glu': '#F7B080',
+  'Phe': '#C7C88D',
+  'Gly': '#C0C0C0',
+  'His': '#D6F6FA',
+  'Lys': '#CEC0F3',
+  'Met': '#C3ED3C',
+  'Asn': '#F29290',
+  'Pro': '#D2D1F8',
+  'Gln': '#F8C4E3',
+  'Arg': '#A6ACEF',
+  'Ser': '#D8B9D4',
+  'Thr': '#F0D6E3',
+  'Trp': '#86B0CC',
+  'Tyr': '#8FC7D1',
```

Comment:
Any reason for duplicating colors? Duplication is typically bad and increases maintenance and probability of mistakes.

Reply:
I think we have two options for the definition of the left/right colors this is one and the other is to translate from TLA to OLA in the color function.

However, I am looking at your feedback and it may not be necessary to change the color component